draft-smith-rats-evidence-trans.md

```diff
@@ -148,52 +142,186 @@ The terminology from CoRIM {{-corim}}, CBOR {{-cbor}}, CDDL {{-cddl}} and COSE {
 
 # Verifier Reconciliation {#sec-verifier-rec}
 
-This specification assumes the reader is familiar with Verifier Reconsiliation as described in {{-corim}}.
+This specification assumes the reader is familiar with Verifier Reconsiliation as described in {{Section 2 of -corim}}.
```

Comment:
@nedmsmith it is reconciliation not Reconsiliation

Reply:
I couldn't find the s spelling in any dictionary. Is it a British spelling? Whatever the spelling it should match what is in the CoRIM draft